Default CI to `slimfast` and add comment control to run other builds. (#669)
napari/docs: docs/developers/contributing/documentation/index.md

on the [napari/docs](https://github.com/napari/docs) GitHub repository.
It's best if you first [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) the [napari/docs](https://github.com/napari/docs) repository to your own GitHub account, create a [feature branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository#creating-a-branch), upload/create/edit files through the GitHub web interface, and then [open a pull request from your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) back to [napari-docs](https://github.com/napari/docs).

> When you submit your PR, a number of jobs will be kicked off, including preview rendering builds of 
> the documentation. By default, they will use the `slimfast` build ("make target"), which is fast, because

Reviewer: ```suggestion
When you submit your PR, a number of jobs will be kicked off, including generation of a preview of 
the documentation. By default, they will use the `slimfast` build ("make target"), which is fast, because
```

> When you submit your PR, a number of jobs will be kicked off, including generation of a preview of 
> the documentation. By default, they will use the `slimfast` build ("make target"), which is fast, because
> it doesn't build any content from outside the `docs` repository and doesn't execute notebook cells.
> This is great for seeing the copy and formatting, but if you want to preview other elements,
> you can trigger more complete builds by commenting on the PR with:

Reviewer 2: ```suggestion
When you submit your PR, CI will kick off several jobs, including generation of a preview of 
the documentation. By default, CI will use the fast `slimfast` build ("make target"), which
doesn't build any content from outside the `docs` repository and doesn't execute example notebook cells.
The `slimfast` build  is great for seeing the copy and formatting. If you want to preview other elements,
you can trigger more complete builds by commenting on the PR with:
```

Author: I worry about confusion with the added "examples": the notebook cells referred to here are not the examples--these are "outside the docs repo"--but rather the cells in the notebooks that make up the docs.
Not sure how to capture this clearly though without being too wordy and needing a TLDR.

Reviewer 2: Maybe replace in my suggestion "and doesn't execute example notebook cells" with "or run notebook cells". Alternatively just strike "doesn't build any content ... notebook cells". The second option may be clearer.

Author: Ok, i riffed on that a bit. Lemme know what you think.

```

> @napari-bot make <target>

Reviewer: well described bot commands 🤖

```
where `<target>` can be:
- `html` : a full build, just like napari.org live
- `html-noplot` : a full build, but without the gallery examples from `napari/napari`
- `docs` : only the content from `napari/docs`, with notebook code cells executed
- `slimfast` : the default, only the content from `napari/docs`, without code cell execution
- `slimgallery` : `slimfast`, but also builds the gallery examples from `napari/napari` 

For more information about these targets see the ["building locally"](build_docs_locally) section 
of the documentation, including the part on [specialized builds](#building-what-you-need).  

> Once the jobs complete—should be fast by default!—you will also be able to [preview the documentation](doc_view_ci) by

Reviewer: ```suggestion
Once the jobs complete you will also be able to [preview the documentation](doc_view_ci) by
```

using the `Check the rendered docs here!` action at the bottom of your PR, which will go to a 
preview website on [CircleCI](https://circleci.com/). Alternatively, you can download a zip file 

> of [the build artifact](#download-documentation-artifact). For a build triggered by a comment, use

Reviewer: ```suggestion
of [the build artifact](#download-documentation-artifact) and open the html files directly in your browser. For a build triggered by a comment, use
```
